package.json

```diff
@@ -47,7 +47,10 @@
     "typescript": "4.3.2"
   },
   "dependencies": {
-    "@superfaceai/ast": "0.0.23",
+    "@superfaceai/ast": "0.0.28",
+    "@superfaceai/parser": "^0.0.19",
+    "@typescript-eslint/eslint-plugin": "^4.27.0",
```

Comment:
Didn't we removed this ESLint deps in some other PR?

Reply:
looks like it's back :trollface: 
i'll fix it